content/articles/getting-started-with-css/index.md

```diff
@@ -0,0 +1,194 @@
+## 1.Selecting Elements
+### Using the Elements' Name
+Developers can select the element uing the element's tag for example `p`,`h2`,`h1` etc as I have shown in the example below.
+```html
+<style>
+   p{}
+</style>
+```
+`p` is the element's name which stands for paragraph.
+
+### Using a Class Name
+A class declaration is done by putting a fullstop before the class name in a style block for example `.red-text`then followed by a declaration block`{}`. Using a class name to select an element will be as shown below.
+```html
+<style>
+   .red-text{}
+</style>
+```
+
+
+### Using an Id Selector
+An element can be selected with a specific Id by writing a `#` character followed by the element's Id. Id attributes are only applicable to one element. An Id attribute has a higher dominance than a class declaration, therefore, prioritized if they collide.An example of an Id selector is like`#Properties-of-CSS`. It can be set in HTML like this;
+```html
+<h2 id= "Properties-of-CSS">
+ ```
+ Using CSS can style Id attributes like this;
+ ```html
+ <style>
+  #Properties-of-CSS{}
+ </style>
+ ```
+
+### Using Inline Styles
+Developers set CSS to a specified HTML element by using the `style` attribute for example;
+```html
+<h2 style=" color: red;">Properties of CSS </h2>
+```
+
+## 2.Doing Various Manipulations to the Elements Selected
+### Using Names of Colors
+With CSS you can change the text. To do this you need to target the element who's text you need to modify with the appropriate selector. You then use the `color`attribute to specify the color you desire for your text as shown below. This can either be done in a separete style sheet or in your HTML file by adding the `style`element like I have in the example.
+```html
+<style>
+ h2{
+     color: red;
+ }
+ </style>
+ ```
+ This will change all the h2 elements to the color red.
+ 
+ Although not advisable in practise,you can also use inline styles to achieve the same results as illustrated in the example below
+ ```html
+ <h2 style="color: red;">Properties of CSS</h2>
+ ```
+ As pointed out above, a class is declared by adding a fullstop(.) accompanied by the class name in a style block.Then followed by a declaration block({}).Using the name of a color to style the class declaration will be like this;
+ ```html
+ <style>
+   .red-text{
+       color: red;
+   }
+ </style>
+ ```
+ In this example,.red-text is the class declaration.
+
+Introducing an Id attribute of Properties-of-CSS and giving it  a background color of black will be as shown in the example below.
+```html
+<style>
+ #Properties-of-CSS{
+    background-color: black;
+ }
+</style>
+```
+### Using Hex Codes
+Hex is the short name for Hexadecimal code. Hex code uses six hexadecimal numbers to specify colors in CSS. Some Hex values of different colors according to [Simmons](http://web.simmons.edu/) are;
```

Comment:
The Simmons link does not go to a resource that aligns with your citation. Update the link to point to the appropriate resource or consider removing the citation all together.